include/fplus/numeric.h

```diff
@@ -135,6 +135,107 @@ X int_power(X base, X exp)
     return base * int_power(base, exp - 1);
 }
 
+template <typename... Conds>
+struct and_ : std::true_type
```

Comment:
@xtofl : I do not know. But @pmalek probably can help. Anyway, It's worth noting that this part was removed from the code anyway by now.